tool/site/client-react/src/components/prescription-tracker/detail/prescriptionDetails.tsx

```diff
@@ -34,34 +33,41 @@ export function createPrescriptionDetailProps(task: Task): PrescriptionDetailPro
   }
 }
 
-export const PrescriptionDetails: React.FC<PrescriptionProps> = ({prescription}) => {
+export const PrescriptionDetails: React.FC<PrescriptionDetailProps> = ({
+  id,
+  type,
+  patientNhsNumber,
+  pharmacy,
+  creationDate,
+  status
+}) => {
   return (
     <SummaryList>
       <SummaryList.Row>
         <SummaryList.Key>ID</SummaryList.Key>
-        <SummaryList.Value>{prescription.id}</SummaryList.Value>
+        <SummaryList.Value>{id}</SummaryList.Value>
       </SummaryList.Row>
       <SummaryList.Row>
         <SummaryList.Key>Type</SummaryList.Key>
-        <SummaryList.Value>{prescription.type}</SummaryList.Value>
+        <SummaryList.Value>{type}</SummaryList.Value>
       </SummaryList.Row>
       <SummaryList.Row>
         <SummaryList.Key>NHS Number</SummaryList.Key>
-        <SummaryList.Value>{prescription.patientNhsNumber}</SummaryList.Value>
+        <SummaryList.Value>{patientNhsNumber}</SummaryList.Value>
+      </SummaryList.Row>
+      {pharmacy &&
+      <SummaryList.Row>
```

Comment:
not that its a big deal, but i missed the `pharmacy &&` initially cause the indentation isnt preserved

Reply:
Fixed